assets/skill_defs.yaml

```diff
@@ -112,3 +112,30 @@ defs:
     item_conditions: []
     stat_effectors:
       - Savagery
+  ReturnAOE:
+    key: ReturnAOE
+    name: Return AOE
+    friendly_name: return_aoe
+    description: Returns 20 damage in AOE for every 5 instances of attacks received.
+    cooldown: 0.0
+    passive: true
+    conditions:
+      - stat_key: AttacksReceived
+        condition:
+          DivisibleBy: 5
```

Comment:
If the leader gets attacked 0 times, then not only will this skill trigger, but it will trigger each tick for as long as the leader doesn't get attacked (which is very probable since it is doing 20 damage per tick in aoe).
For the first issue, you can add a stat condition with a minimum value.
For the second, a cooldown should be good enough to fix the issue for now.

Reply:
That was a big oversight that I am glad got caught. I have implemented your suggested fixes.